gradle/dependencies.gradle

```diff
@@ -39,6 +39,8 @@ ext {
             mockito                          : "1.10.19",
             hamcrest                         : "1.3",
             systemRules                      : "1.19.0",
-            logcaptor                        : "2.5.0"
+            logcaptor                        : "2.5.0",
+
+            legacySdk                        : "2.6.0"
```

Comment:
azureLegacySdk? don't want to get confused with our legacy sdk :)

Reply:
this is our legacy sdk, maybe I should rename to `legacySdkForSmokeTests`?

Reply:
i think legacySdk is fine then.. thought it was azure :(
or aiJavaLegacySdk to make it more specific?